File: NomadMapApp/composeApp/src/androidMain/kotlin/org/nomad/mapapp/data/model/Address.kt
package org.nomad.mapapp.data.model

import com.google.android.gms.maps.model.LatLng
import kotlinx.serialization.Serializable

@Serializable
data class Address(
    val id: Int = 0,
    val location: String,
    val lat: Float,
    val lng: Float,
    val companyId: Int = 0
) {
    fun toLatLng(): LatLng = LatLng(lat.toDouble(), lng.toDouble())
}

File: RazorPage/RazorPage/Models/Address.cs
namespace RazorPage.Models
{
    public class Address
    {
        public int Id { get; set; }
        public string Location { get; set; }
        public float Lat { get; set; }
        public float Lng {  get; set; }
        public int CompanyId { get; set; }
    }
}
